src/Auth0.OidcClient.Core/Auth0SecureClientOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Auth0.OidcClient
+{
+    /// <summary>
+    /// A version of <see cref="Auth0ClientOptions"/> designed for clients that can keep
```

Comment:
This wording doesn't feel strong enough to me. If you're using a client secret in this case, you're probably acting like a confidential client on a public client. If you're doing that, you're doing the wrong thing (public clients are always used by a person so there should always be a user involved). You should be calling calling an API and that API should do that action on your behalf.

> If your app is publically distributed should probably not be using this.

No probably about it :)

Reply:
Maybe rename this from `Auth0SecureClientOptions` to `Auth0ConfidentialClientOptions` ?

Reply:
Too subtle ... and also not true. Using a client secret does not make you a confidential client. Might be misleading.

Reply:
Sorry this file was supposed to be in a different PR. Will address that there.